### PR TITLE
Add GitHub Actions

### PR DESCRIPTION
**Pull request by jeff-hernandez**
This PR will integrate GitHub Actions with PyPI to help automate the release process. After publishing a release on GitHub, a GitHub Action will upload the package to PyPI.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='composeml',
-    version='0.1.3',
+    version='0.1.4',
```
Comment (jeff-hernandez) [inline setup.py]: I bumped versions for demo purposes. I do plan to revert these changes.

**Comment by kmax12** [comment]
This looks good!
